src/api/objects/ObjectAPI.js

```diff
@@ -215,6 +227,188 @@ define([
         return utils.makeKeyString(identifier);
     };
 
+    let migrations = [
```

Comment:
Maybe we should move these into separate source files?

Reply:
I'll consider it when writing the plugin.